build/Targets/VSL.Settings.targets

```diff
@@ -316,7 +315,7 @@
       $(BuildNumberSuffix.Split('.')[1].PadLeft(2,'0'))
     </BuildNumberPart2>
 
-    <NuGetReleaseVersion>$(RoslynSemanticVersion)</NuGetReleaseVersion>
+    <NuGetReleaseVersion>1.1.1</NuGetReleaseVersion>
```

Comment:
Note that this is wrong for Master, but I don't have a better way of doing this for stabilization.